docs/concepts/cluster-administration/sysctl-cluster.md

```diff
@@ -93,7 +93,11 @@ flag of the kubelet, e.g.:
 ```shell
 $ kubelet --experimental-allowed-unsafe-sysctls 'kernel.msg*,net.ipv4.route.min_pmtu' ...
 ```
+For minikube, this can be done via the `extra-config` flag:
 
+ ```shell
+ $ minikube start --extra-config="kubelet.AllowedUnsafeSysctls=kernel.msg*,net.ipv4.route.min_pmtu"...
+ ```
 Only _namespaced_ sysctls can be enabled this way.
```

Comment:
This command is not rendering correctly in the [preview](https://deploy-preview-5897--kubernetes-io-master-staging.netlify.com). But I don't understand why. I'll investigate.